src/sentry/conf/server.py

```diff
@@ -2260,6 +2260,14 @@ def build_cdc_postgres_init_db_volume(settings):
             "volumes": {"redis": {"bind": "/data"}},
         }
     ),
+    "redis-cluster": lambda settings, options: (
+        {
+            "image": "grokzen/redis-cluster:7.0.10",
+            "ports": {f"700{idx}/tcp": f"700{idx}" for idx in range(6)},
+            "volumes": {"redis-cluster": {"bind": "/redis-data"}},
+            "environment": {"IP": "0.0.0.0"},
```

Comment:
see ~ https://github.com/Grokzen/docker-redis-cluster#important-for-mac-users ~ why it's needed.